### ISSUE TITLE
Delete physical image file

### ISSUE THREAD

**Issue by miyconst** [Polyjuice/Images#12]
The physical image file should be deleted when image updated or removed.

**Comment by miyconst** [comment]
Now files are being deleted when:
1. Illustration being deleted.
2. Illustration file being replaced.

Files will stay when:
1. User drags different files into same are without saving the object.

After dropping/selecting a file it will be uploaded and saved automatically. Server remembers only one *oldImageUrl* which will be removed on save, all other files will stay. Still need some improvements.

**Comment by warpech** [comment]
Can this be finished?

**Comment by miyconst** [comment]
Yes.

**Comment by miyconst** [comment]
Now all old files will be removed after image saved, but the files will keep staying if user closes page without saving data.

Need answer for this issue to finish: https://github.com/Starcounter/Starcounter/issues/2697

**Comment by miyconst** [comment]
Because no one was able to tell me how to execute some action when user left page, I used another approach.

Now old files, which don't have `Content` object in database will be deleted on `Images` app startup. This is prbably better, because even working `SessionEnd` or `PageLeave` event would not serve all of the situations. For exampe: user opens Images app, uploads a file and then server lost power. The uploaded file will live on the server forever.

Closing the issue for now.
